dds/src/builtin_topics.rs

```diff
@@ -81,3 +231,63 @@ pub struct SubscriptionBuiltinTopicData {
     pub topic_data: TopicDataQosPolicy,
     pub group_data: GroupDataQosPolicy,
 }
+
+impl DdsType for SubscriptionBuiltinTopicData {
+    fn type_name() -> &'static str {
+        "SubscriptionBuiltinTopicData"
+    }
+}
+
+impl<'de> DdsDeserialize<'de> for SubscriptionBuiltinTopicData {

```

Comment:
All this DdsDeserialize implementations are new? Shall I check them? Or write compatibility and/or integration tests? I see "only" tests for look up, are those testing this deserialization as well?

Reply:
They are not new but simply a copy of the already existing ones for the composed discovery types. Unfortunately I found that with our current implementation it is not really possible to use this deserialize inside the other deserialize. This is something that will need to be handled separately.